modules/translations.mjs

```diff
@@ -42,7 +42,7 @@ export function getDiscordLocale(langCode) {
 
 i18next.init({
     lng: 'en-US',
-    debug: true, // process.env.NODE_ENV === 'development',
+    debug: process.env.NODE_ENV !== 'ci',
```

Comment:
Not 100% sure what your intent is on this line. If you are looking for a way to check if it is being run in CI (GitHub Actions) then the env var `CI` will always be set to `"true"`

Reply:
See example [here](https://github.com/the-hideout/tarkov-dev/blob/f4bcadf329b11547675061aec7e9bcd56e856c5a/scripts/get-contributors.js#L58)

Reply:
As the changes in index.mjs also show, when we manually run the command registration, we also set NODE_ENV to "ci".

Reply:
Basic idea is we don't need i18next to barf out a lot of debug text when the commands are being registered.